panoramisk/testing.py

```diff
@@ -45,14 +45,14 @@ class Manager(manager.Manager):
 
     def __init__(self, **config):
         self.defaults.update(
-            connection_class=Connection,
```

Comment:
I renamed connection_class to protocol_factory to follow on https://docs.python.org/3/library/asyncio-eventloop.html#creating-connections